blocks/article-header/article-header.css

```diff
@@ -0,0 +1,19 @@
+.block .article-byline p {
+  display: inline;
+  margin-right: 20px;
+  font-size: 14px;
+  font-weight: 400;
+}
+
+
+.article-byline a:any-link {
+  font-family: var(--ff-gilroy-bold);
+  background-image: linear-gradient(to right,var(--link-background-color) 50%,transparent 50%);
+  background-size: 205% 2px;
+  background-repeat: no-repeat;
+  background-position: bottom right;
+}
+
+.article-byline p > a:hover {
```

Comment:
same here, we could replace https://github.com/hlxsites/servicenow/blob/main/styles/styles.css#L236 to not duplicate